app/templates/administrative-units/new.hbs

```diff
@@ -0,0 +1,204 @@
+{{page-title "Voeg bestuurseenheid toe"}}
+{{breadcrumb "Voeg bestuurseenheid toe"}}
+
+<div class="au-c-body-container au-c-body-container--scroll">=
+  <PageHeader class="au-u-margin au-u-margin-bottom-large">
+    <:title>Voeg bestuurseenheid toe</:title>
+  </PageHeader>
+
+  <form class="au-o-box au-o-flow au-o-flow--large" aria-label="administrative-unit-creation-form"
+  {{on "submit" (perform this.createAdministrativeUnitTask)}}>
+    <EditCard>
+      <:title>Bestuurseenheid</:title>
+      <:card as |Card|>
+        <Card.Columns>
+          <:left as |Item|>
+            <Item @labelFor="administrative-unit-name">
+              <:label>Naam eredienst</:label>
+              <:content>
```

Comment:
I just saw comment number 284 in the Figma saying to delete the _eredienst_ part so the page would be more generic.